src/components/authentication/Success.js

```diff
@@ -10,6 +10,7 @@ const Success = (props) => {
 
   // save token to localStorage
   useEffect(() => {
+    console.log('REACHED');
```

Comment:
We can get rid of this console.log unless you still need it.

Reply:
removed

Reply:
Great